stories/bar/2_label_value.tsx

```diff
@@ -61,6 +61,7 @@ export const Example = () => {
         fontStyle: 'normal',
         padding: 0,
         fill: color('value color', '#000'),
```

Comment:
I've been testing and I think for the example white fill color looks better with a dark shadow:
 
```suggestion
        fill: color('value color', '#fff'),
```

<img width="1207" alt="Screenshot 2020-09-28 at 12 29 58" src="https://user-images.githubusercontent.com/2750668/94427217-a0cd8100-0186-11eb-88db-3f24273a7c95.png">

Reply:
I also increased the size fo the value font size to 12.

Reply:
Do you think it should be a default setting or just a better starting point for this example?

Reply:
I was seeing the other charts where we have labels/values on top and normally they are black or with `textInvertible: true`

<img width="1551" alt="Screenshot 2020-09-28 at 14 42 08" src="https://user-images.githubusercontent.com/2750668/94439893-e1ce9100-0198-11eb-87e0-9e8b2582f397.png">


So for consistency, I think we should keep the **value labels black** and just increase the size to **12**. 

How do you feel of then creating a second example to show that the value label can be customized? In this new example, you would use **white value labels** with a **shadow**.

Reply:
@miukimiu which would you prefer?

1. Changing color contrast of the shadow
2. Changing color contrast of the text itself
3. Either of the above depending on if a shadow is enabled

Reply:
From my understanding when it flips the color on text the shadow should change too.
But you are right, there is also the additional scenario where the shadow is not enabled (`transparent`?) and only the text changes.

Reply:
Yeah thats true but I was actually thinking that the shadow be `null` or `undefined`, as it's not needed when the text contrast is dynamic.